docs/other/install-gwms-frontend.md

```diff
@@ -698,7 +700,7 @@ There are a few things that can be checked prior to submitting user jobs to HTCo
         user@host $ condor_status -any
         MyType               TargetType           Name
         glideresource        None                 MM_fermicloud026@gfactory_inst
-        Scheduler            None                 fermicloud020.fnal.gov
+        Scheduler            None                 fermicloud0wms-renew-proxies20.fnal.gov
```

Comment:
This looks like an accidental change?

Reply:
@mambelli ?

Reply:
Yes, it should be "fermicloud020.fnal.gov". The hostname could be anything in theory but seems that "wms-renew-proxies" has been pasted in the middle of it.